docs/understand/sizing-guide.md

```diff
@@ -368,8 +511,21 @@ Here are a few notable metrics to keep track of when sizing lakeFS:
 
 `gateway_request_duration_seconds` - Histogram of latency per [S3 Gateway](architecture.md#s3-gateway) operation.
 
-`go_sql_stats_*` - Important client-side metrics collected from the PostgreSQL driver.
-See [The full reference here](https://github.com/dlmiddlecote/sqlstats#exposed-metrics){: target="_blank" }.
+<div class="tabs">
```

Comment:
Seems like another layout issue around this area

Reply:
I can't see it, I put the tabs on purpose so we can use it when we'll add kv metrics to Postgres

Reply:
I see - can you just change the order of the tabs to be - postgres - dynamodb, like all the other sections?